met-web/src/components/engagement/banner/BannerWithImage.tsx

```diff
@@ -77,16 +77,27 @@ const BannerWithImage = ({ savedEngagement, children }: BannerProps) => {
                             <Editor editorState={getEditorState(rich_description)} readOnly={true} toolbarHidden />
                         </Grid>
                         <Grid item xs={12}>
-                            <MetHeader2 style={{ fontWeight: 600 }} color="black">
-                                {`Engagement dates: ${formatDate(start_date, 'MMMM dd, yyyy')} to ${formatDate(
-                                    end_date,
-                                    'MMMM dd, yyyy',
-                                )}`}
-                            </MetHeader2>
+                            {isSmallScreen ? (
+                                <MetHeader3 style={{ fontWeight: 600 }} color="black">
+                                    {`Engagement dates: ${formatDate(start_date, 'MMMM dd, yyyy')} to ${formatDate(
+                                        end_date,
+                                        'MMMM dd, yyyy',
+                                    )}`}
+                                </MetHeader3>
+                            ) : (
+                                <MetHeader2 style={{ fontWeight: 600 }} color="black">
+                                    {`Engagement dates: ${formatDate(start_date, 'MMMM dd, yyyy')} to ${formatDate(
+                                        end_date,
+                                        'MMMM dd, yyyy',
+                                    )}`}
+                                </MetHeader2>
+                            )}
                         </Grid>
                         <Grid item xs={12}>
                             <Stack direction="row" spacing={1}>
-                                <Typography variant="subtitle1">status:</Typography>
+                                <Typography sx={{ fontWeight: 800 }} variant="subtitle1">
```

Comment:
why specifically fontWeight: 800?

Reply:
change it to bold for consistency across application